README.md

```diff
@@ -14,12 +14,25 @@ More information is available on the
 
 ## Installation
 
+Prerequisites
+-------------
+* Have the most recent version of Java (Java 1.8)
+* Have Scala downloaded
```

Comment:
We probably want to recommend whatever version of Scala is listed for the most recent ScalaTion distribution.